src/libraries/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/Http2Connection.cs

```diff
@@ -1525,11 +1557,24 @@ private void WriteHeaders(HttpRequestMessage request, ref ArrayBuffer headerBuff
                 {
                     WriteBytes(KnownHeaders.ContentLength.Http2EncodedName, ref headerBuffer);
                     WriteLiteralHeaderValue("0", valueEncoding: null, ref headerBuffer);
+                    headerListSize += 14 + HeaderField.RfcOverhead;
```

Comment:
Would avoiding magic numbers by KnownHeaders.ContentLength.Name.Length or something hit the perf? 🙂

Reply:
It shouldn't. I'll update it to the more readable version.